tests/network/probes.go

```diff
@@ -227,6 +228,28 @@ var _ = SIGDescribe("[ref_id:1182]Probes", func() {
 			Entry("[test_id:TODO]with working Exec probe", createExecProbe(period, initialSeconds, timeoutSeconds, "uname", "-a"), blankIPFamily),
 		)
 
+		Context("guest agent ping", func() {
+			BeforeEach(func() {
+				vmi = libvmi.NewFedora(withMasqueradeNetworkingAndFurtherUserConfig(
+					withLivelinessProbe(createGuestAgentPingProbe(period, initialSeconds)),
+				)...)
+				vmi = tests.VMILauncherIgnoreWarnings(virtClient)(vmi)
+				By("Waiting for agent to connect")
+				tests.WaitAgentConnected(virtClient, vmi)
+				Expect(console.LoginToFedora(vmi)).To(Succeed())
+			})
+
+			It("[test_id:TODO] VM stops when guest agent is disabled", func() {
+				Expect(stopGuestAgent(vmi)).To(Succeed())
```

Comment:
nit: I would move this to the test setup block, leaving only the expectation (eventually) in the test's body.

Reply:
I think it is part of the test, not setup. The test checks that if guest agent is disabled, then the VM will stop.

Reply:
My point exactly: if the test checks that when the guest agent is disabled, the VM will stop, the test should consist exactly of that expectation.

Anyway, this is just a nit - you can ignore it.